tests/debian/Dockerfile

```diff
@@ -2,8 +2,8 @@ FROM qserv/stack:debian_latest
 MAINTAINER Fabrice Jammes <fabrice.jammes@in2p3.fr>
 
 USER root
-# MariaDB dependencies
-RUN apt-get install -y git cmake libncurses5-dev liblzma-dev zlib1g-dev
+# eups/eupspkg and MariaDB dependencies
+RUN apt-get update && apt-get install -y git patch cmake libncurses5-dev liblzma-dev zlib1g-dev
```

Comment:
I often pass `-qq` to `apt-get` in scripts to cut down on the rather verbose default output.